src/utils/azureUtils.ts

```diff
@@ -89,6 +89,7 @@ const azExtDisplayInfo: Partial<Record<AzExtResourceType, AzExtResourceTypeDispl
     AppServiceKubernetesEnvironment: { displayName: localize('containerService', 'App Service Kubernetes Environment') },
     AppServicePlans: { displayName: localize('serverFarms', 'App Service plans') },
     AppServices: { displayName: localize('webApp', 'App Services') },
+    ArcEnabledMachines: { displayName: localize('arcEnabledMachines', 'Azure Arc-enabled machines') },
```

Comment:
@andyleejordan  would it be okay to remove the Azure part from the display name? We are trying to be consistent about naming. https://github.com/microsoft/vscode-azureresourcegroups/issues/806

This isn't a release blocker at all.

Reply:
I asked about that and was told that "Arc is always prefaced by Azure because the trademark is the combo of the two words."